src/misc/base32/base32_decode.c

```diff
@@ -57,7 +59,7 @@ int base32_decode(const unsigned char *in,  unsigned long inlen,
    LTC_ARGCHK(in     != NULL);
    LTC_ARGCHK(out    != NULL);
    LTC_ARGCHK(outlen != NULL);
-   LTC_ARGCHK(alphabet_idx < 4);
+   LTC_ARGCHK(alpha_id < 4);
```

Comment:
as it's an `enum` now and `enum`'s are normally mapped to `int`'s I guess it should be like
```
LTC_ARGCHK(alpha_id >= BASE32_RFC4648);
LTC_ARGCHK(alpha_id <= BASE32_CROCKFORD);
```